packages/i18n/src/locales/en_US.ts

```diff
@@ -683,6 +683,11 @@ const translations: Catalog = {
               link: 'https://docs.mongodb.com/manual/reference/method/db.collection.getShardDistribution',
               description: 'Prints the data distribution statistics for a sharded collection.',
               example: 'db.coll.getShardDistribution()'
+            },
+            analyze: {
+              link: '',
```

Comment:
I think we've added links here in the past, even if they 404'd, knowing that they would all follow the same pattern for collection methods.